SPECS/mariner-rpm-macros/macros

```diff
@@ -52,6 +52,9 @@
 %{?mariner_ccache_enabled:CCACHE_DIR="/ccache-dir" ; export CCACHE_DIR ;}\
 %{nil}
 
+# use zstd compression for binary package payloads
+%_binary_payload w7T0.zstdio
```

Comment:
Do we want to set `%_source_payload` as well? Looks like that affects .src.rpm compression.

Reply:
I briefly looked at changing the `%_source_payload`, but the source files do not compress very well, so I think the benefit would be minimal at best.

Reply:
I performed a local build of "make input-srpms" with `%_source_payload w7T0.zstdio`. The size of the src.rpm files was nearly identical. Does not seem worth it to change this parameter.

For example, `kernel-5.15.116.1-2.cm2.src.rpm`, which is around 187 MB, shrunk by only 234 KB.
kernel-5.15.116.1-2.cm2.src.rpm - gzip 9 cpio   -  195699714 bytes
kernel-5.15.116.1-2.cm2.src.rpm - zstd 7T0 cpio -  195459770 bytes